content/6_Advanced/Multiplicative.mdx

```diff
@@ -41,28 +41,25 @@ _Dirichlet Convolution and Möbius Inversion_
 3. Möbius inversion is also a discussion regarding $g(n) = \sum_{d|n} f(d)$ , but it does not require $f$ to be multiplicative and is applicable in cases where $g(n)$ is known and $f(n)$ is to be determined. Since $I*\mu = e$， $g*\mu = f*I*\mu = f*e = f$，and $f(n) = \sum_{d|n} g(d) \cdot \mu\left(\frac{n}{d}\right)$. Similarly,  $g(n) = \sum_{n|d} f(d) \Rightarrow f(n) = \sum_{n|d} g(d) \cdot \mu\left(\frac{d}{n}\right)$，Binomial inversion is also a similar technique. An example illustrates the relationship between the Euler function and the Möbius function: since $\sum_{d|n} \varphi(d) = n$，then $\varphi(n) = \sum_{d|n} \mu(d) \cdot \frac{n}{d}$，which implies $\frac{\varphi(n)}{n} = \sum_{d|n} \frac{\mu(d)}{d}$
 
 ### Focus problem # 1
-<FocusProblem problem="tutorial" />  
+<FocusProblem problem="tutorial" />
 
 _Approach:_
 
-First a review on linear sieve, this will be used in both focus problem 1 and 3: 
+First a review on linear sieve, this will be used in both focus problem 1 and 3:
 <LanguageSection>
 <CPPSection>
 ```cpp
-void get_prime(int n)
-{
-    for( int i = 2 ; i <= n; i++ )
-    {
-        if(!st[i]) primes[++cnt] = i;
-        for( int j = 1 ; primes[j] <= n / i ; j++ )
-        {
-            st[i * primes[j]] = true;
-            if( i % primes[j] == 0 ) break; 
-        }
-    }
+void get_prime(int n) {
```

Comment:
for sake of consistency, could you use the sieve impl [here](https://usaco.guide/gold/divisibility#solution---counting-divisors)

or just have a link like "hey, if you forgor sieve, here's a link to impl again"